pyfstat/make_sfts.py

```diff
@@ -291,7 +291,7 @@ def get_base_template(self, i, Alpha, Delta, h0, cosi, psi, phi, F0, F1, F2, tre
 Freq = {:1.18e}
 f1dot = {:1.18e}
 f2dot = {:1.18e}
-refTime = {:10.6f}"""
+refTime = {:1.9f}"""
```

Comment:
10.9 ???

Reply:
```
>>> a = 123456789.123456789
>>> print("{:10.9f}".format(a))
123456789.123456791
>>> print("{:1.9f}".format(a))
123456789.123456791
>>> a = 1.123456789
>>> print("{:10.9f}".format(a))
1.123456789
>>> print("{:1.9f}".format(a))
1.123456789
```
This is odd... I though  that number represented the number of ensured places...?

Reply:
> I though that number represented the number of ensured places...?

For `f`, it's actually the number of formatted characters summing up pre- and post-decimals, +1 for the `.` itself:
```
print("{:20.9f}".format(a))
 123456789.123456791
```
(notice leading blank space).

Since I don't think the `.cff` format cares for blanks, `{:1.9f}` should do just as well, or even just `{:.9f}`.